docs/cockpit/setup.md

```
@@ -0,0 +1,212 @@
+an actionable error pointing at the install path for your OS.
+
+### Verify
+
+```bash
+aoe cockpit doctor
+```
+
+Sample output on a machine where Claude is installed but the others
+aren't:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)